ironfish-cli/src/commands/status.ts

```diff
@@ -40,14 +40,20 @@ export default class Status extends IronfishCommand {
     const screen = blessed.screen({ smartCSR: true, fullUnicode: true })
     const statusText = blessed.text()
     screen.append(statusText)
+    let previousResponse: GetNodeStatusResponse | null = null
 
     // eslint-disable-next-line no-constant-condition
     while (true) {
       const connected = await this.sdk.client.tryConnect()
 
       if (!connected) {
         statusText.clearBaseLine(0)
-        statusText.setContent('Node: STOPPED')
+        if (previousResponse) {
+          statusText.setContent(renderStatus(previousResponse, flags.all))
+          statusText.insertTop("Node: Disconnected \n")
```

Comment:
We don't need the extra space here since were just inserting a new line

```suggestion
          statusText.insertTop("Node: Disconnected\n")
```

Reply:
Also there is a bug at the bottom of this file where an extra new line is inserted.